testing/btest/plugins/zeek-version-link.zeek

```diff
@@ -1,5 +1,4 @@
 # @TEST-DOC: Poking at internals: Expect an undefined zeek_version_X_Y_Z_plugin symbol in the plugin's .so/.dynlib. If this test turns out to be brittle, remove it, but we lost the mechanism.
-# @TEST-REQUIRES: nm --version
```

Comment:
This will likely break on Windows. Of course, the test is probably already broken on Windows for other reasons, but this will break it more.

Reply:
I thought there are no plugins on windows anyhow?

Reply:
Yeah, I guess plugins are completely broken as is. Ignore this.